### PR TITLE
Bug 1833558: create cluster roles for modiyfing scc-s

### PR DESCRIPTION
**Pull request by soltysh**
/assign @stlaz

### CODE REVIEW

File: manifests/0000_20_kube-apiserver-operator_00_cr-anyuid.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: "true"
+  creationTimestamp: null
```
Comment (stlaz) [inline manifests/0000_20_kube-apiserver-operator_00_cr-anyuid.yaml:6]: nit: remove creationTimestamp in the yamls

Comment (soltysh) [reply manifests/0000_20_kube-apiserver-operator_00_cr-anyuid.yaml]: :man_facepalming: fixed.

**Comment by soltysh** [comment]
@stlaz ptal again

**Comment by soltysh** [comment]
/retest
/assign @deads2k 
for approval

**Comment by soltysh** [comment]
This was rebased to hopefully fix the failures, tagging it back.

**Pull request merged**